src/client/components/PrivateRoute.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  const isLoggedin = localStorage.getItem("token");
```

Comment:
aren't you supposed to check of it's a valid token to assure that the user is actually logged in?

Reply:
We don't actually have an endpoint for this at the moment. But all of the API endpoints except for the one to login are protected, so we discussed barely checking the existence of a token would at least stop the user from going into the application without having access to any content.